apps/explorer/lib/sync.ts

```diff
@@ -7,3 +7,16 @@ export function getIsSynced(data?: { prevTimestamps: string[] }) {
         Date.now() - hoursInMilliseconds(12)
     : false
 }
+
+export function getIsIndexing(
+  consensusState?: { index: { height: number } },
+  blockMetrics?: { index: { height: number } }
+) {
+  if (!blockMetrics || !consensusState) {
+    return false
+  }
+
+  // Only show indexing warnings if the node is more than 2 blocks behind on indexing.
```

Comment:
Is 2 blocks sufficient? I guess it would be unusual but we often mine a bunch of blocks in short succession, not sure how long indexing a block takes...

Reply:
I was thinking it would be enough to prevent the index warning from popping up every block during normal operation, and in rare cases where it does drop behind more than 2, its actually good to show the warning. I also suspect the indexing will be fast enough that this will not happen, but if that ends up wrong we can adjust this value.